### PR TITLE
[JIT] SVE Cleanup: Merging SVE_AE_3A, SVE_AN_3A, SVE_EP_3A, SVE_ER_3A, SVE_ET_3A, SVE_EU_3A into SVE_AA_3A

### PR DESCRIPTION
**Pull request by TIHan**
Merging more formats into SVE_AA_3A.

### CODE REVIEW

**Comment by TIHan** [comment]
@dotnet/arm64-contrib @kunalspathak this is ready.

**Review by a74nh — APPROVED** [review]
LGTM

File: src/coreclr/jit/emitfmtsarm64sve.h [hunk 1]
```diff
@@ -140,7 +140,6 @@ IF_DEF(SVE_2BS, IS_NONE, NONE) // Instruction has 2  possible encoding types, ty
 IF_DEF(SVE_AA_3A,   IS_NONE, NONE) // SVE_AA_3A  ........xx...... ...gggmmmmmddddd  -- SVE bitwise logical operations (predicated)
 IF_DEF(SVE_AB_3B,   IS_NONE, NONE) // SVE_AB_3B  ................ ...gggmmmmmddddd  -- SVE integer add/subtract vectors (predicated)
 IF_DEF(SVE_AC_3A,   IS_NONE, NONE) // SVE_AC_3A  ........xx...... ...gggmmmmmddddd  -- SVE integer divide vectors (predicated)
-IF_DEF(SVE_AE_3A,   IS_NONE, NONE) // SVE_AE_3A  ........xx...... ...gggmmmmmddddd  -- SVE integer multiply vectors (predicated)
```
Comment (kunalspathak) [inline src/coreclr/jit/emitfmtsarm64sve.h:143]: can you append the comments of the formats that you are removing (and in previous PR as well) and put it next to `AA_3A`, so they don't get lost?

File: src/coreclr/jit/emitarm64sve.cpp [hunk 2]
```diff
@@ -9903,17 +9897,17 @@ BYTE* emitter::emitOutput_InstrSve(BYTE* dst, instrDesc* id)
     switch (fmt)
     {
         // Scalable.
-        case IF_SVE_AA_3A: // ........xx...... ...gggmmmmmddddd -- SVE bitwise logical operations + integer add/subtract
-                           // vectors + SVE integer min/max/difference (predicated)
+        case IF_SVE_AA_3A: // ........xx...... ...gggmmmmmddddd -- SVE bitwise logical operations + SVE integer
```
Comment (kunalspathak) [inline src/coreclr/jit/emitarm64sve.cpp:9900]: i would do this change at the definition and just remove the description from here (for the ones you have already merged).

**Review by kunalspathak — APPROVED** [review]

File: src/coreclr/jit/emitfmtsarm64sve.h (hunk 1 above)
Comment (TIHan) [reply src/coreclr/jit/emitfmtsarm64sve.h:143]: Yea, can do that.

File: src/coreclr/jit/emitarm64sve.cpp (hunk 2 above)
Comment (TIHan) [reply src/coreclr/jit/emitarm64sve.cpp:9900]: Ok, so just remove these descriptions for the ones we merged, got it.

**Pull request merged**